app/helpers/rat_adoptions_helper.rb

```diff
@@ -0,0 +1,2 @@
+module RatAdoptionsHelper
```

Comment:
since there are no helpers in here, how would you feel about removing this file for now?